docs/reference/modules/http.asciidoc

```diff
@@ -36,7 +36,7 @@ from the outside. Defaults to the actual port assigned via `http.port`.
 
 |`http.publish_host` |The host address to publish for HTTP clients to connect to. Defaults to `http.host` (if set) or `network.publish_host`.
 
-|`http.host` |Used to set the `http.bind_host` and the `http.publish_host` Defaults to `http.host` or `network.host`.
+|`http.host` |Used to set the `http.bind_host` and the `http.publish_host` Defaults to `network.host`.
```

Comment:
Technically it defaults to an empty list, but it only has an effect if set (see previous two settings) so I don't think we need to talk about the default at all:

```suggestion
|`http.host` |Used to set the `http.bind_host` and the `http.publish_host`.
```